server/routes/applications.js

```diff
@@ -30,6 +30,7 @@ router.get('(/manage-applications)?', (req, res, next) => {
       test: listTest.data,
       future: listFuture.data
     };
+    console.log('GET LIST ALL: ', viewData.this_data.test);
```

Comment:
I will remove this

Reply:
You just left that in to test us of course. ;)